app/views/handlers/block.py

```diff
@@ -13,17 +13,21 @@
 
 
 class BlockHandler:
-    def __init__(self, schema, questionnaire_store, language, current_location):
+    def __init__(
+        self, schema, questionnaire_store, language, current_location, return_to
+    ):
         self._schema = schema
         self._questionnaire_store = questionnaire_store
         self._language = language
         self._current_location = current_location
+        self._return_to = return_to
         self.block = self._schema.get_block(current_location.block_id)
 
```

Comment:
What's this break in variable initialisation for? It seems to denote a logical seperation between the two blocks of vars, but I don't see one.

Reply:
If you are referring to the extra line on line 25, I'm not sure, it wasn't introduced as part of this pr.